a3-live/src/components/TeamInterface.js

```diff
@@ -7,68 +7,88 @@ import FormControl from '@material-ui/core/FormControl';
 import ListItemText from '@material-ui/core/ListItemText';
 import Select from '@material-ui/core/Select';
 import Checkbox from '@material-ui/core/Checkbox';
+import {OutlinedInput} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
 
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    card: {
+        maxWidth: 345,
+    },
+}));
+/*
+const inputLabel = React.useRef(null); // TODO: Lookup in docs
+const [labelWidth, setLabelWidth] = React.useState(0);
+React.useEffect(() => {
+        setLabelWidth(inputLabel.current.offsetWidth);
+    }, []);
+*/
 
 function TeamInterface(props) {
-    const [TeamLead, setTeamLead] = React.useState({});
+    const classes = useStyles();
     const [TeamMembers, setTeamMembers] = React.useState([]);
```

Comment:
It is safe to initialize `TeamMembers` state to `props.Team.Employees` - values are ids, that will be parsed by our `Select` perfectly